java/src/main/java/org/rocksdb/SstFileWriter.java

```diff
@@ -25,12 +25,38 @@ public void open(final String filePath) throws RocksDBException {
     open(nativeHandle_, filePath);
   }
 
+  @Deprecated
```

Comment:
Can you add the javadoc please? Including @deprecated annotation in the javadoc itself?